recipes/entity.rb

```diff
@@ -0,0 +1,53 @@
+#
+# Cookbook Name:: cloud_monitoring
+# Recipe:: entity
+#
+# Configure the cloud_monitoring_entity LWRP to use the existing entity
+# for the node by matching the server IP.
+#
+# Copyright 2014, Rackspace
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# cm is defined in libraries/cloud_monitoring.rb
+class Chef::Recipe
+  include Opscode::Rackspace::Monitoring
+end
+
+cm(defined?(node['cloud_monitoring']['rackspace_api_key']) ? node['cloud_monitoring']['rackspace_api_key'] : nil,
+   defined?(node['cloud_monitoring']['rackspace_username']) ? node['cloud_monitoring']['rackspace_username'] : nil,
+   defined?(node['cloud_monitoring']['rackspace_auth_url']) ? node['cloud_monitoring']['rackspace_auth_url'] : nil)
+
+response = cm.list_entities.body
+
+
+response["values"].each do |value|
+  unless value["ip_addresses"].nil? || node["cloud"].nil?
```

Comment:
I'd have gone with something like `next if value["ip_addresses"].nil? || node["cloud"].nil?`, so that the next 3 lines aren't indented another level.